lib/src/key/mod.rs

```diff
@@ -4,26 +4,28 @@
 /// crate::key::root::hb                 /!hb{ts}/{nd}
 /// crate::key::root::nd                 /!nd{nd}
 /// crate::key::root::ni                 /!ni
-/// crate::key::root::ns                 /!ns{ns}
+/// crate::key::root::ns                 /!ns{ns name}
 ///
 /// crate::key::node::all                /${nd}
 /// crate::key::node::lq                 /${nd}!lq{lq}{ns}{db}
 ///
 /// crate::key::namespace::all           /*{ns}
-/// crate::key::namespace::db            /*{ns}!db{db}
-/// crate::key::namespace::di            /+{ns id}!di
+/// crate::key::namespace::db            /*{ns}!db{db name}
+/// crate::key::namespace::di            /*{ns}!di
 /// crate::key::namespace::lg            /*{ns}!lg{lg}
+/// crate::key::namespace::ns            /*{ns}!ns [new]
```

Comment:
`ns` in `*{ns}` is now u32. This `/*{ns}!ns` is for storing the DefineNamespaceStatement for that instance of namespace, not the namespace alias.

Reply:
Ah, great idea! Maybe do that in next PR, or would you mind doing it @tobiemh if you're going to cut beta.10 today?